### PR TITLE
HTTPException Error, Response Body Example added

### PR DESCRIPTION
**Pull request by shivambhatia5432**
**Description**
- I have added Response Examples for few requests.
- Added HttpException instead of Internal Server Error if user not found
- Request at user/user/:userID changes to user/:userID
- Was not able to use get a request for userID in Swagger UI, now it is working after adding its the type i.e. string

**Type of Change:**

- [x] Code
- [x] New Feature

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

**Screenshots**
   Original                 |   Updated
  :------------------------:|---------------------:
Screenshot 2021-03-27 at 1 55 13 AM | Screenshot 2021-03-27 at 1 53 37 AM

### CODE REVIEW

**Comment by shivambhatia5432** [comment]
@Abhishek-kumar09 Please have a look at this PR, I will be happy to make changes if there are any

File: src/user/user.service.ts [hunk 1]
```diff
@@ -17,7 +17,16 @@ export class UserService {
   // Get a single User
   async getUser(UserID): Promise<User> {
     const User = await this.UserModel.findById(UserID).exec();
-    return User;
+    if (User) {
```
Comment (Abhishek-kumar09) [inline src/user/user.service.ts:20]: Please also rename `User` to `user`

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Revert changes of package-lock.json since no more additions in package

**Comment by shivambhatia5432** [comment]
@Abhishek-kumar09 Changes requested implemented

**Comment by Abhishek-kumar09** [comment]
Thanks @shivambhatia5432

**Pull request merged**